flake-info/src/data/export.rs

```diff
@@ -73,6 +73,7 @@ pub enum Derivation {
         package_outputs: Vec<String>,
         package_default_output: Option<String>,
         package_programs: Vec<String>,
+        package_mainProgram: Option<String>,
```

Comment:
This top of this file says changes to this file need a bump of the `/VERSION` to make sure a fresh index will be created. Is that not needed for this PR because the change is 'backwards-compatible enough' or an oversight?